src/frontend/src/pages/PIAForm/index.tsx

```diff
@@ -577,8 +604,9 @@ const PIAFormPage = () => {
               ? PiaStatuses.EDIT_IN_PROGRESS
               : pia?.status,
         });
-        if (pia?.id) {
-          navigate(getEditPath(pathname));
+        if (pia?.id && !pathname?.split('/').includes('review')) {
+          const regex = /(?<=\/)view/g;
+          navigate(pathname.replace(regex, 'edit'));
```

Comment:
please use geteditpath. And this is fixed and tested in 

    [UTOPIA-1279](https://apps.itsm.gov.bc.ca/jira/browse/UTOPIA-1279)

Reply:
this code will break if there is a link called 'preview'. the same way review is breaking by replacing review to reedit. The geteditPath breaks the url properly and generates the edit link